states/menu.py

```diff
@@ -5,22 +5,29 @@
 
 
 class MenuOptions(StateOptions):
+    """Enum values for options available in the menu"""
+
     BATTLE = 1
     TEAM = 2
     EXIT = 3
 
 
 class Menu(State):
+    """Displays, holds, and reads options for Menu state change"""
+
     option: Optional[MenuOptions] = None
 
     def __init__(self):
+        """Initializes the menu state and lists available options checking against user input if exit is not chosen"""
         super().__init__()
         while self.option is not MenuOptions.EXIT:
             print(TEXT["MAIN"]["ENTRY"])
             self.option = self.check_input(MenuOptions)
             self.choose_option()
 
     def choose_option(self):
+        """Runs respective state chosen"""
```

Comment:
Consider changing to 'Enters next game state based on player choice'

Reply:
Considered changing to `Enters next game state based on player choice` and did so